src/Features/LanguageServer/Protocol/Features/Options/CompletionOptionsStorage.cs

```diff
@@ -88,6 +82,10 @@ public static CompletionOptions GetCompletionOptions(this IGlobalOptionService o
     public static readonly Option2<bool> ForceExpandedCompletionIndexCreation
         = new(nameof(CompletionOptions), nameof(ForceExpandedCompletionIndexCreation), defaultValue: false);
 
+    public static readonly PerLanguageOption2<bool?> ShowNewSnippetExperience
```

Comment:
necessary for https://github.com/dotnet/roslyn/pull/64804/files#diff-31303c421816ee9001acbb872ec31e6ef3d701cd04eba6f2a41d02e49d70ae21R10504